digdag-standards/src/main/java/io/digdag/standards/command/DockerCommandExecutor.java

```diff
@@ -70,6 +70,9 @@ private Process startWithDocker(Path projectPath, TaskRequest request, ProcessBu
         }
         else {
             imageName = baseImageName;
+            if (dockerConfig.has("pull-always")) {
```

Comment:
@komamitsu 

I fixed it (I was misunderstanding about `build` config so please ignore my previous comment.).

Then, i have one more suggestion.
i think better of `pull-always`'s default is true because in most cases it is convenient to be enabled.

Hou do you think it?